rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/schedule/ScheduleAdminController.java

```diff
@@ -379,4 +382,58 @@ public ResponseEntity<?> scheduleSourceUploadForReleaseComponents()throws TExcep
         HttpStatus status = HttpStatus.ACCEPTED;
         return new ResponseEntity<>(requestStatus, status);
     }
+
+    @Operation(
+            summary = "Check the status of a scheduled service.",
+            description = "Checks whether a specific service is scheduled for execution.",
+            tags = {"Admin"}
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Service status returned successfully.",
+                    content = @Content(mediaType = "application/json",
+                            schema = @Schema(implementation = Map.class,
+                                    example = "{\"serviceName\": \"exampleService\", \"isScheduled\": true}"))),
+            @ApiResponse(responseCode = "400", description = "Bad request due to missing or invalid service name.",
+                    content = @Content(mediaType = "application/json",
+                            schema = @Schema(implementation = String.class,
+                                    example = "Service name is required"))),
+    })
+    @RequestMapping(value = SCHEDULE_URL + "/status", method = RequestMethod.GET)
+    public ResponseEntity<Map<String, Object>> checkServiceStatus(
+            @Parameter(description = "Name of the service", required = true)
+            @RequestParam String serviceName
+    ) throws TException {
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+
+        if (serviceName.isBlank()) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Service name is required");
+        }
```

Comment:
If not sent, `serviceName` can be `null`. Better use CommonUtils
```suggestion
        if (CommonUtils.isNullEmptyOrWhitespace(serviceName)) {
            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Service name is required");
        }
```

Reply:
I have addressed the above mentioned changes. Please review